stories/basic.stories.tsx

```diff
@@ -50,6 +70,49 @@ const BasicForm = () => {
           ]}
         />
       </Form.Item>
+      <Form.List name="testArray" rules={[rule as any]}>
+        {(fields, { add }, { errors }) => {
+          return (
+            <div>
+              {fields.map((field) => (
+                <>
+                  <Form.Item
+                    label="Name"
+                    initialValue={""}
+                    name={[field.name, "name"]}
+                    rules={[rule]}
+                  >
+                    <Input />
+                  </Form.Item>
+                  <Form.Item>
```

Comment:
`FormItem` isn't needed

Reply:
As you can see in antd documentation in order to add nested Lists you need to wrap it with a Form.Item also, otherwise the nested items verification doesn't work
<img width="1232" alt="Screenshot 2024-02-16 at 08 15 46" src="https://github.com/MrBr/antd-zod/assets/59712160/25e269b0-939c-4b34-b119-c154fcfec436">

Reply:
Thanks for clarification